File: java/Tugasb.java
public class Tugasb {
	// atribut private
    private String Nama;
	private String NegaraAsal;
	private String TahunBerdiri;
	private String[] Pemain;

	// setter/getter
	public void setNama(String Nama) {
		this.Nama = Nama;
	}
	public String getNama() {
		return Nama;
	}
	public void setNegaraAsal(String NegaraAsal) {
		this.NegaraAsal = NegaraAsal;
	}
	public String getNegaraAsal() {
		return NegaraAsal;
	}
	public void setTahunBerdiri(String TahunBerdiri) {
		this.TahunBerdiri = TahunBerdiri;
	}
	public String getTahunBerdiri() {
		return TahunBerdiri;
	}
	public void setPemain(String[] Pemain) {
		this.Pemain = Pemain;
	}
	public String getPemain(int i) {
		return Pemain[i];
	}
}

File: php/tugas b/tugasb.php
<?php

class Timbola{
    private $Nama = "";
    private $NegaraAsal = "";
    private $TahunBerdiri = "";
    private $Pemain = array();

	public function __construct($Nama = "", $NegaraAsal = "", $TahunBerdiri = "", $Pemain = array()){
        $this->Nama = $Nama;
        $this->NegaraAsal = $NegaraAsal;
        $this->TahunBerdiri = $TahunBerdiri;
        $this->Pemain = $Pemain;
	}

    public function setNama($Nama){
		$this->Nama = $Nama;
	}
    public function getNama(){
		return $this->Nama;
	}

    public function setNegaraAsal($NegaraAsal){
		$this->NegaraAsal = $NegaraAsal;
	}
    public function getNegaraAsal(){
		return $this->NegaraAsal;
	}

    public function setTahunBerdiri($TahunBerdiri){
		$this->TahunBerdiri = $TahunBerdiri;
	}
    public function getTahunBerdiri(){
		return $this->TahunBerdiri;
	}

    public function setPemain($Pemain){
		$this->Pemain = $Pemain;
	}
    public function getPemain($i){
		return $this->Pemain[$i];
	}
}

?>